src/app/record-editor/search-bar/search-bar.component.html

```diff
@@ -1,17 +1,21 @@
 <div class="navbar-form input-group col-md-7 pull-left inline">
-  <input class="form-control search" [(ngModel)]="query" (keyup.enter)="searchOrGo()" placeholder="Search records"
-  />
+  <input class="form-control search" [(ngModel)]="query" (keyup.enter)="searchOrGo()" placeholder="Search records" />
   <span class="input-group-btn">
     <a class="btn btn-default fa fa-search" (click)="searchOrGo()"></a>
   </span>
 </div>
 <div *ngIf="resultCount >= 0" class="navbar-form col-md-5 inline">
-  <ng-container *ngIf="resultCount > 0; else nothingFoundTemplate">
-    <i class="btn btn-default fa fa-chevron-left" (click)="onPreviewClick()"></i>
-    <span class="result-text">{{cursor + 1}} / {{resultCount}}</span>
-    <i class="btn btn-default fa fa-chevron-right" (click)="onNextClick()"></i>
+  <ng-container [ngSwitch]="resultCount">
+    <ng-container *ngSwitchCase="0">
+      <span class="result-text form-control-static">Nothing found</span>
```

Comment:
just a small improvement in English, show `No results` instead of `Nothing found`